app/src/pages/homePage/homePage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { useNavigator } from '@kibalabs/core-react';
```

Comment:
added this page so `/` redirects to `/domains` automatically. once we have auth the router can take care of it but until then good to have this i think

Reply:
yh lol, i was thinking the same. either this or login 🤷